.gitignore

```diff
@@ -4,5 +4,6 @@
 .env
 .DS_Store
 venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore virtual environment directories. Ensure that this is intentional and consistent with your project's setup, especially if you are using `.venv` as your virtual environment directory.